plasmapy/physics/distribution.py

```diff
@@ -143,6 +143,144 @@ def Maxwellian_1D(v,
         return distFunc
 
 
+def Maxwellian_velocity_2D(vx,
```

Comment:
This doesn't seem to be appearing in the built docs: https://1504-46810954-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/physics/distribution.html

Reply:
Do I have to manually add stuff in the docs?

Reply:
Looks like it's showing up now
https://1508-46810954-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/physics/distribution.html?highlight=distribution#module-plasmapy.physics.distribution